src/crate_metadata.rs

```diff
@@ -58,12 +65,12 @@ impl CrateMetadata {
         let mut original_wasm = target_directory.clone();
         original_wasm.push("wasm32-unknown-unknown");
         original_wasm.push("release");
-        original_wasm.push(package_name.clone());
+        original_wasm.push(lib_name.clone());
         original_wasm.set_extension("wasm");
 
         // {target_dir}/{package_name}.wasm
         let mut dest_wasm = target_directory.clone();
-        dest_wasm.push(package_name.clone());
+        dest_wasm.push(lib_name.clone());
```

Comment:
Note that `package_name` will still be used for the `contract` bundle, which could be confusing when viewing the `target` dir if they are different. 

We could just replace the `package_name` field `CrateMetadata` with something like `contract_artifact_name`, and populate with the `lib` name.

Reply:
Good point, I updated the PR.